recipes/aws-lambda-powertools/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "aws-lambda-powertools" %}
+{% set version = "1.10.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aws_lambda_powertools-{{ version }}.tar.gz
+  sha256: 795316eea0b5b37d335ad75c5fe6a279d75b9021ed72cff2e4f63184c316440b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry >=0.12
+    - python >=3.6,<3.9
+  run:
+    - aws-xray-sdk >=2.5.0,<3
+    - boto3 >=1.12,<2
+    - python-fastjsonschema >=2.14.5,<3
+    - jmespath >=0.10.0,<0.11
+    - python >=3.6,<3.9
+  run_constrained:
+    - email-validator
```

Comment:
`email-validator` was listed as an optional dependency (https://github.com/awslabs/aws-lambda-powertools-python/blob/2f840bfbaa3390ddaa81aa2b34f17832e4ef5e3c/pyproject.toml#L28) but with no specific version. 
I'd therefore exclude it again, correct?

Reply:
Yup, it won't do anything if it's not got a version constraint associated

Reply:
Ok will do, thanks for the confirmation! Any idea why the added it in the first place?